init.ts

```diff
@@ -370,14 +375,16 @@ const VSCODE_EXTENSIONS = JSON.stringify(vscodeExtensions, null, 2) + "\n";
 
 if (useVSCode) {
   await Deno.writeTextFile(
-    join(directory, ".vscode", "extensions.json"),
+    join(resolvedDirectory, ".vscode", "extensions.json"),
     VSCODE_EXTENSIONS,
   );
 }
 
-const manifest = await collect(directory);
-await generate(directory, manifest);
+const manifest = await collect(resolvedDirectory);
+await generate(resolvedDirectory, manifest);
 
+// Specifically print unresolvedDirectory, rather than resolvedDirectory in order to
+// not leak personal info (e.g. `/Users/MyName`)
 console.log("\n%cProject created!", "color: green; font-weight: bold");
-console.log("\`cd my-project\` to enter to the project directory.");
+console.log(`\`cd ${unresolvedDirectory}\` to enter to the project directory.`);
```

Comment:
**Alternate approach: Why not just print `Deno.args[0]` here?**

I think that would make the concern of reading from command line args less DRY. "reading from command line args" feels like one of those areas where we can easily make an impactful mistake, so it felt important to optimize for DRY-ness there.